Form/DataTransformer/IdToEntityTransformer.php

```diff
@@ -2,23 +2,25 @@
 
 namespace Opifer\ContentBundle\Form\DataTransformer;
 
-use Doctrine\Common\Collections\ArrayCollection;
-use Doctrine\Common\Collections\Collection;
+use Opifer\ContentBundle\Model\ContentManager;
+use Opifer\ContentBundle\Model\ContentManagerInterface;
 use Symfony\Component\Form\DataTransformerInterface;
-use Symfony\Component\Form\Exception\TransformationFailedException;
 
 class IdToEntityTransformer implements DataTransformerInterface
 {
-    protected $entityManager;
+    /**
+     * @var ContentManager
+     */
+    protected $contentManager;
 
     /**
      * Constructor
      *
-     * @param object $entityManager
+     * @param ContentManagerInterface $contentManager
      */
-    public function __construct($entityManager)
+    public function __construct(ContentManagerInterface $contentManager)
```

Comment:
I'm fine with this change, but then I think the classname should be changed to `IdToContentTransformer` since it's not usable for other entities anymore.